app/assets/stylesheets/config/_generator.scss

```diff
@@ -294,8 +294,8 @@
     true
   ),
   (
-    'top',
-    'top',
+    'to',
```

Comment:
This was just to test that it picks up CSS changes. @Zhao-Andy will revert this.